scribejava-apis/src/main/java/com/github/scribejava/apis/HiOrgServerApi20.java

```diff
@@ -0,0 +1,52 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.oauth2.clientauthentication.ClientAuthentication;
+import com.github.scribejava.core.oauth2.clientauthentication.RequestBodyAuthenticationScheme;
+
+/**
+ * OAuth2 API for HiOrg-Server
+ *
+ * @see <a href="https://wiki.hiorg-server.de/admin/oauth2">HiOrg-Server OAuth API documentation</a>
+ */
+public class HiOrgServerApi20 extends DefaultApi20 {
+
+    private final String version;
+
+    protected HiOrgServerApi20() {
+        this("v1");
+    }
+
+    protected HiOrgServerApi20(String version) {
+        this.version = version;
+    }
+
+    private static class InstanceHolder {
+
+        private static final HiOrgServerApi20 INSTANCE = new HiOrgServerApi20();
+    }
+
+    public static HiOrgServerApi20 instance() {
+        return InstanceHolder.INSTANCE;
+    }
+
+    public static HiOrgServerApi20 customVersion(String version) {
+        return new HiOrgServerApi20(version);
+    }
+
+    @Override
+    public String getAccessTokenEndpoint() {
+        return "https://www.hiorg-server.de/api/oauth2/" + version + "/token.php";
+    }
+
+    @Override
+    protected String getAuthorizationBaseUrl() {
+        return "https://www.hiorg-server.de/api/oauth2/" + version + "/authorize.php";
+    }
+
+    @Override
+    public ClientAuthentication getClientAuthentication() {
+        return RequestBodyAuthenticationScheme.instance();
+    }
```

Comment:
HiOrg-Server does work without this override.
And according to the https://tools.ietf.org/html/rfc6749#section-2.3.1
default is better.
"...the authorization server MAY support including the client credentials in the request-body... 
Including the client credentials in the request-body using the two
   parameters is NOT RECOMMENDED and SHOULD be limited to clients unable
   to directly utilize the HTTP Basic authentication scheme (or other
   password-based HTTP authentication schemes).
```